x-pack/plugins/infra/public/containers/logs/log_summary/bucket_size.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useMemo } from 'react';
+
+export function useBucketSize(
+  startTimestamp: number | null,
+  endTimestamp: number | null
+): number | null {
+  const bucketSize = useMemo(() => {
+    if (!startTimestamp || !endTimestamp) {
+      return null;
+    }
+    return (endTimestamp - startTimestamp) / 100;
```

Comment:
I checked the previous code. The existing name was `LOAD_BUCKETS_PER_PAGE`. I can reuse it but to be honest it doesn't tells me much.

Do you know/remember the context behind it? Maybe we can find a better name. Otherwise I restore that one

Reply:
With the scrollable minimap it dictated into how many buckets the minimap was split. Now something like `SUMMARY_BUCKET_COUNT` or so would probably do it?

Reply:
Perfect 👍